src/main/java/com/cydeo/model/ClientVendor.java

```diff
@@ -1,6 +1,32 @@
 package com.cydeo.model;
 
-public class ClientVendor {
```

Comment:
Thanks for review, I did try to do so, but apparently my knowledge about resolving conflict is short, would you please led me to how to do it ( I don't see it in my local ( after merging Master Local, and can't edit it here in Giyhub.  Thanks

Reply:
I did some research, apparently, I should see "Resolve Conflict" button, that I can't see (It may because Resolve conflict button has been removed for me ( who do the pull request,)